css/style.css

```diff
@@ -198,7 +198,188 @@ ul {
     background-color: #f7033c;
 }
 
+.container__footer {
+    height: 130px;
+}
+
 .inline {
     font-size: 14px;
     line-height: 1.2;
+}
+
+@media (max-width: 1024px) {
+    .title {
+        font-size: 42px;
+        margin-top: 25px;
+    }
+
+    .top__container {
+        height: 565px;
+    }
+
+    .top__image {
+        background-position: top;
+        background-size: contain;
+    }
+
+    .top::after {
+        height: 488px;
+    }
+}
+
+@media (max-width: 768px) {
+    .container {
+        width: 100%;
+        padding-left: 16px;
+    }
+
+    .logo {
+        font-size: 28px;
+    }
+
+    .list__item + .list__item {
+        margin-left: 20px;
+    }
+
+    .title {
+        font-size: 30px;
+        margin: 25px 0 0 16px;
+        width: 340px;
+    }
+
+    .top__container {
+        width: 100%;
+        height: 425px;
+    }
+
+    .top::after {
+        height: 368px;
+    }
+}
+
+@media (max-width: 640px) {
+    .list__link {
+        font-size: 14px;
+    }
+
+    .logo {
+        margin-right: 20px;
+    }
+
+    .info__title {
+        font-size: 24px;
+    }
+
+    .info__content {
+        width: 100%;
+        font-size: 18px;
+    }
+
+    .team__block {
+        width: 180px;
+        height: 180px;
+    }
+
+    .avatar {
+        width: 100px;
+        height: 100px;
+    }
+
+    .list__forma {
+        padding: 0 20px;
+    }
+
+    .inline {
+        font-size: 12px;
+    }
+}
+
+@media (max-width: 480px) {
+    .container {
+        display: table;
+        height: 60px;
+    }
+    
+    .logo {
+        justify-content: flex-start;
+    }
+
+    .list {
+        justify-content: flex-start;
+    }
+
+    .list__link--small--screen {
+        display: none;
+    }
+
+    .top::after {
+        height: 230px;
+    }
+
+    .top__container {
+        height: 270px;
+    }
+
+    .title {
+        font-size: 20px;
+        width: 225px;
+    }
+
+    .info__title--line::after {
+        top: 35px;
+    }
+
+    .team__block {
+        width: 150;
+        height: 150px;
+    }
+
+    .team__block + .team__block {
+        margin-left: 10px;
+    }
+
+    .avatar {
+        width: 80px;
+        height: 80px;
+    }
+
+    .name {
+        margin-top: 0;
+        font-size: 13px ;
+    }
+
+    .list__forma {
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .forma + .forma {
+        margin: 30px 0 0 0 ;
```

Comment:
extra white space

Reply:
I dont quite understand what you mean about extra white space in this place

Reply:
margin: 30px 0 0 **0 ;** => margin: 30px 0 0 0;